tests/debugging/test_uploader.py

```diff
@@ -52,6 +52,9 @@ def test_uploader_full_buffer():
         with pytest.raises(BufferFull):
             uploader._encoder.put(item)
```

Comment:
Couldn't you just put a larger item? Still if this works to overload the buffer that seems fine.

Reply:
In theory the test is size-independent because we compute the number of times we can encode a certain message based on the size of a single message. So, in theory, this test should have never failed in its original form. If it keeps failing, the problem must be somewhere else.